### PR TITLE
Techdebt/milli 659

### PR DESCRIPTION
**Pull request by Phebonacci**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,27 +35,42 @@
...
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
+    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-syntax-import-meta": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^9.0.0",
+    "babel-jest": "^25.1.0",
```
Comment (leamarielabagala) [inline package.json]: Hi @Phebonacci! Just a q, do we still need these or can we use the @babel/ version of these packages?

Comment (Phebonacci) [reply package.json]: Hi @leamarielabagala ! I haven't seen a @babel/jest npm package yet so I kept this instead.

File: package.json [hunk 2]
```diff
@@ -35,27 +35,42 @@
...
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
+    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-syntax-import-meta": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (leamarielabagala) [inline package.json]: Hi @Phebonacci ! is this the same with @babel/core?

Comment (Phebonacci) [reply package.json]: Hi @leamarielabagala ! Not entirely. I looked it up and it's a bridge between babel 6 yo 7 (https://github.com/babel/babel-bridge). I just used npx babel upgrade and that's what it gave. lol

Comment (leamarielabagala) [reply package.json]: ohh okie2 hehe

**Pull request merged**